### PR TITLE
Add label studio app tutorial

### PR DESCRIPTION
**Pull request by jackklpan**
Signed-off-by: Jack Pan <jackpan@infuseai.io>

### CODE REVIEW

File: docs/primehub-app-tutorial-label-studio.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+![](assets/app_tutorial_labelstudio_login_page.png)
+
+It will open a new window and show the Label Studio UI. You can find your login information by clicking `Manage` in the `Apps` tab and then clicking the eyes icon. The `$(PRIMEHUB_GROUP)` is the group name.
+
+![](assets/app_tutorial_labelstudio_login_info.png)
+
+## Label Dataset
+### What we need?
+
+- The dataset in PrimeHub you want to label (we use `/datasets/dog-demo` in this tutorial)
```
Comment (even-wei) [inline docs/primehub-app-tutorial-label-studio.md:39]: Suggest to add hyper link at dataset to https://docs.primehub.io/docs/next/guide_manual/admin-dataset

File: docs/primehub-app-tutorial-label-studio.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- The directory in group volume that you want to save the labeled results (we use `/project/label-studio/dog-demo-labeled` in the tutorial)
```
Comment (even-wei) [inline docs/primehub-app-tutorial-label-studio.md:40]: Replace `/project/label-studio/dog-demo-labeled` with `/project/<group_name>/dog-demo-labeled`

File: docs/primehub-app-tutorial-label-studio.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    ![](assets/app_tutorial_labelstudio_create.png)
+
+2. Enter your `Project Name`. Skip the `Data Import` step. And choose the `Labeling Setup`. Here we choose `Semantic Segmentation with Polygons`.
+
+3. Delete the original `Labels` settings and `Add` our own label names.
+
+    ![](assets/app_tutorial_labelstudio_labeling_setup.png)
+
+4. Click the `Settings` on the upper-right. Click `Cloud Storage` and `Add Source Storage` to sync the `/datasets/dog-demo` dataset to label. You need to set `Local path` to `/datasets/dog-demo` and click `Sync Storage`.
```
Comment (even-wei) [inline docs/primehub-app-tutorial-label-studio.md:56]: Turn on toggle of Treat every bucket object as a source file

**Comment by jackklpan** [comment]
All fixed. Thanks!

**Comment by even-wei** [comment]
LGTM 👍

**Pull request merged**